XamlControlsGallery/ControlPages/ExpanderPage.xaml

```diff
@@ -0,0 +1,83 @@
+<!--
+//*********************************************************
+//
+// Copyright (c) Microsoft. All rights reserved.
+// THIS CODE IS PROVIDED *AS IS* WITHOUT WARRANTY OF
+// ANY KIND, EITHER EXPRESS OR IMPLIED, INCLUDING ANY
+// IMPLIED WARRANTIES OF FITNESS FOR A PARTICULAR
+// PURPOSE, MERCHANTABILITY, OR NON-INFRINGEMENT.
+//
+//*********************************************************
+-->
+<Page x:Class="AppUIBasics.ControlPages.ExpanderPage" xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+      xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" xmlns:local="using:AppUIBasics"
+      xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+      xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006" 
+      xmlns:contract7Present="http://schemas.microsoft.com/winfx/2006/xaml/presentation?IsApiContractPresent(Windows.Foundation.UniversalApiContract,7)"
+      xmlns:muxc="using:Microsoft.UI.Xaml.Controls"
+      mc:Ignorable="d">
+    <StackPanel>
+        <local:ControlExample x:Name="Example1" HeaderText="An Expander with text in the header and content areas">
+            <local:ControlExample.Example>
+                <muxc:Expander x:Name="Expander1" IsExpanded="False" ExpandDirection ="Down" Header="This text is in the header" Content="This is in the content"/>
```

Comment:
Does the Content default to HorizontalElignment = Center? Maybe an odd choice if that's the case?

Reply:
Could you elaborate on why this is odd or what the answer here should be instead?

Reply:
Purely subjective :) Putting myself in an app developer's shoes, I suspect I would want my content to be left aligned more often than any other alignment. I expect this to be easy to overwrite...and that just about everyone would overwrite the center alignment. HorizontalElignment = Left just feels like a better default to me. Again, chances are, apps will be putting a container like Grid or StackPanel as the first child of the content in real world scenarios. I'm just guessing the apps would want these containers to be left aligned because that results in a more predictable layout instead of potentially unexpectedly large padding.

Reply:
Thanks for the explanation! I'll include this in the API discussion